lib/reek/context/code_context.rb

```diff
@@ -136,24 +132,30 @@ def matches?(candidates)
       end
 
       def full_name
-        exp.full_name(context ? context.full_name : '')
+        exp.full_name(parent ? parent.full_name : '')
```

Comment:
Yep, way clearer already. Before that i would **always** think "what the hell is context here again?!", scroll up, down, left and right and the be like "ah, yeah, its just the friggin parent" :(